zfmt/ast.go

```diff
@@ -316,6 +316,9 @@ func (c *canon) decl(d ast.Decl) {
 	case *ast.ConstDecl:
 		c.write("const %s = ", d.Name)
 		c.expr(d.Expr, "")
+	case *ast.TypeDecl:
```

Comment:
Nit: Put this after OpDecl.

Reply:
We need to at some point go through and alphabetize ast nodes elsewhere.